tests/garage/torch/misc/test_tensor_utils.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+import pytest
+import tensorflow as tf
+import torch
+
+import garage.tf.misc.tensor_utils as tf_utils
+import garage.torch.misc.tensor_utils as torch_utils
+from tests.fixtures import TfGraphTestCase
+
+
+class TestTensorUtil(TfGraphTestCase):
```

Comment:
in the future i'll definitely request a test which uses known inputs/outputs rather than just comparing output to some other function, but this will suffice for this PR.